core/src/main/java/bio/terra/pearl/core/model/participant/EnrolleeRelation.java

```diff
@@ -18,6 +18,7 @@
 @SuperBuilder
 @NoArgsConstructor
 public class EnrolleeRelation extends BaseEntity {
+    private UUID id;
```

Comment:
I don't think this field is explicitly needed here, since it's part of the `BaseEntity` class that EnrolleeRelation extends